src/tests/stub_SquidConfig.cc

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright (C) 1996-2023 The Squid Software Foundation and contributors
+ *
+ * Squid software is distributed under GPLv2+ license and includes
+ * contributions from numerous individuals and organizations.
+ * Please see the COPYING and CONTRIBUTORS files for details.
+ */
+
+#include "squid.h"
+
+#include "SquidConfig.h"
+
+InstanceIdDefinitions(SquidConfig, "config");
```

Comment:
If this code survives, let's keep it in sync with the other change request.

```suggestion
InstanceIdDefinitions(SquidConfig, "cfg");
```

Reply:
I removed this file.